examples/asset-upload/src/ExampleAssetUploadBlock.tsx

```diff
@@ -5,11 +5,11 @@ import { useAssetUpload, useBlockAssets, useFileInput } from '@frontify/app-brid
 import { Button } from '@frontify/fondue';
 import '@frontify/fondue-tokens/styles';
 import { BlockProps } from '@frontify/guideline-blocks-settings';
-import { FC, useEffect, useState } from 'react';
+import { useEffect, useState } from 'react';
 import 'tailwindcss/tailwind.css';
 import { IMAGE_SETTING_ID } from './settings';
 
-export const ExampleAssetUploadBlock: FC<BlockProps> = ({ appBridge }) => {
+export const ExampleAssetUploadBlock = ({ appBridge }: BlockProps) => {
```

Comment:
```suggestion
export const ExampleAssetUploadBlock = ({ appBridge }: BlockProps): ReactElement => {
```

Reply:
Good point, going to add this, thanks! 👍🏻